vmm/src/seccomp_filters.rs

```diff
@@ -688,6 +688,9 @@ fn vmm_thread_rules(
         (libc::SYS_wait4, vec![]),
         (libc::SYS_write, vec![]),
         (libc::SYS_writev, vec![]),
+        #[cfg(target_arch = "x86_64")]
```

Comment:
These are in alphabetical order - in your commit message you say `libc::kill` uses these syscalls. Please can you show me where?

Reply:
The order is fixed. I trace the CLH process twice with `strace` and get the following messages:

```
[pid 1923324] --- SIGSYS {si_signo=SIGSYS, si_code=SYS_SECCOMP, si_call_addr=0x7f70f3ce03f9, si_syscall=__NR_lstat, si_arch=AUDIT_ARCH_X86_64} ---

[pid 2021984] --- SIGSYS {si_signo=SIGSYS, si_code=SYS_SECCOMP, si_call_addr=0x7f5e8cff05c2, si_syscall=__NR_getcwd, si_arch=AUDIT_ARCH_X86_64} ---
```

Reply:
Thanks. In your commit message you claim it comes from `libc::kill` did you have a backtrace showing that or?

Reply:
I also trace the CLH process with `strace` and kill CLH by executing `kill` in the console. The seccomp violations disappear.